### PR TITLE
update terraform code

### PR DESCRIPTION
**Pull request by JoseMotto-sg**
## Change description

Updates terraform code to match latest version

### Development

- [x] Lint rules pass locally
- [x] Application changes have been tested thoroughly
- [x] Automated tests covering modified code pass

### Security

- [x] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review 

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

File: .terraform-version [hunk 1]
```diff
@@ -1 +1 @@
-1.2.8
+1.6.3
```
Comment (jheison-rodriguez) [inline .terraform-version:1]: same comments as for the PR https://github.com/StratusGrid/terraform-aws-limits-monitor/pull/5

File: .tflint.hcl [hunk 2]
```diff
@@ -1,5 +0,0 @@
-plugin "aws" {
```
Comment (jheison-rodriguez) [inline .tflint.hcl:1]: Why was that file removed?

**Review by jheison-rodriguez — CHANGES_REQUESTED** [review]
same

File: .tflint.hcl (hunk 2 above)
Comment (JoseMotto-sg) [reply .tflint.hcl:1]: Because our workflow-config takes care of that file, and we don't need a special tflint in this case

**Review by jheison-rodriguez — APPROVED** [review]
Approved as per internal conversation of leaving those versions like that
